### DISCUSSION TITLE
make expanded details panel not clickable

### DISCUSSION THREAD

**Discussion by danwillo** [KevinVandy/material-react-table#1135] [Q&A]
Hi, i have expanded details from a row which is clickable. would anyone know how to make the details panel not clickable? 

thanks

**Comment by KevinVandy** [comment]
With css

**Reply by danwillo** [reply]
ive been trying to do this with pointer-events... cant seem to make it work unfortunately. both inline and in my css. any suggestions?

**Reply by danwillo** [reply]
all good figured it out. 

added the following:

.Mui-TableBodyCell-DetailPanel 

{
  pointer-events: none;
}
